### ISSUE TITLE
DeepSORT w/ TF OD API?

### ISSUE THREAD

**Issue by Nannigalaxy** [nwojke/deep_sort#264]
Hi,

How to add support for tf od api models? particularly for tf2 version.
Can I modify to use `tf.saved_model.load` function to load to generate detections?
If there is any implementation regarding this please guide me.

Thank you.

**Comment by wandyu** [comment]
Yes please. I am trying to add the IDs for my object detection model. I feel this deep sort algorithm would help. Please let me know if you have any solution.
